src/SIL.XForge.Scripture/ClientApp/src/assets/i18n/checking_ar.json

```diff
@@ -242,7 +242,7 @@
     "questions": "أسئلة"
   },
   "checking_questions": {
-    "no_questions_found": "No questions found to match your filter"
+    "no_questions_found_filtered": "No questions found to match your filter"
```

Comment:
This isn't wrong or going to break anything, but I want to make sure you're aware you don't have to update the non-English localization files. The English files get uploaded to Crowdin, and the non-English ones are downloaded and overwrite the files in the repo, so the update can easily get applied without going through file by file.

And before the files get updated, it just falls back to the English strings.